twing/webpack-options.js

```diff
@@ -5,12 +5,18 @@ const twingVirtualEnvironmentPath = require.resolve(
 
 const twingLoader = {
   test: /\.twig$/,
-  use: {
-    loader: 'twing-loader',
-    options: {
-      environmentModulePath: twingEnvironmentNodePath,
+  use: [
+    // Webpack loaders run in reverse, so this top loader will run last
```

Comment:
Thanks for the comment, @calebeby! Does that mean we want/need this loader to run last? In other words, if a new loader gets added in the future, should we ensure the new loader is below this loader? If so, let's also leave a note about this detail as well, thanks! 🙂

Reply:
It has to run after the twing loader, but not necessarily last. It sort of depends on what another loader is doing

Reply:
I put that comment there not to say that this specific loader has to run _last_, but to remind me or other people that webpack's ordering is opposite of what you'd expect (I forget literally every time)

Reply:
> I put that comment there not to say that this specific loader has to run last, but to remind me or other people that webpack's ordering is opposite of what you'd expect (I forget literally every time)

I also forget every time, so this note is very much appreciated!